### PR TITLE
Flex page

### PR DESCRIPTION
**Pull request by TomWoodward**

### CODE REVIEW

File: src/app/pages/flex-page/blocks/CardsBlock.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      position: relative;
+      z-index: 1;
+    }
+
+    &.card_style_rounded > .content-block-card {
+      border-radius: 1rem;
+      border: thin solid ui-color(black);
+
+      /* is this the right color for default background? */
+      background-color: #fff;
```
Comment (RoyEJohnson) [inline src/app/pages/flex-page/blocks/CardsBlock.scss:25]: Pattern library colors are defined here:
https://github.com/openstax/pattern-library/blob/master/core/pattern-library/colors.scss
You could use `ui-color(default)` or `ui-color(white)`

File: src/app/pages/flex-page/blocks/HeroBlock.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    width: 100%;
+    margin: 0;
+
+    > div.hero-inner-wrapper {
+        /* to show above divider images */
+        position: relative;
+        z-index: 1;
+
+        display: grid;
+        grid-template: 1fr / 1fr 1fr;
```
Comment (RoyEJohnson) [inline src/app/pages/flex-page/blocks/HeroBlock.scss:13]: For mobile: `grid-template-columns: 1fr` and `text-align: center`
hero-content should have `max-width: $text-content-max` and `margin: auto`

Comment (TomWoodward) [reply src/app/pages/flex-page/blocks/HeroBlock.scss:13]: @RoyEJohnson this comment isn't necessary anymore right? you added the mobile switch?

**Review by RoyEJohnson — APPROVED** [review]
All looks good to me.

**Pull request merged**